java-sdk/src/main/java/com/netflix/conductor/sdk/executor/WorkflowExecutor.java

```diff
@@ -0,0 +1,370 @@
+        }
+        if(taskRunner != null) {
+            taskRunner.shutdown();
+        }
+        ses.shutdown();
+    }
+
+    /**
+     * Starts a workflow execution and waits till
+     * @param name
+     * @param version
+     * @param input
+     * @return
+     */
+    public Workflow executeWorkflow(String name, int version, Map<String, Object> input) {
+        StartWorkflowRequest request = new StartWorkflowRequest();
+        request.setInput(input);
+        request.setName(name);
+        request.setVersion(version);
+
+        String workflowId = workflowClient.startWorkflow(request);
+        CountDownLatch latch = new CountDownLatch(1);
+        runningWorkflows.put(workflowId, latch);
+        Uninterruptibles.awaitUninterruptibly(latch);
+        runningWorkflows.remove(workflowId);
+        return workflowClient.getWorkflow(workflowId, true);
+    }
+
+
+
+    TypeReference<List<TaskDef>> listOfTaskDefs = new TypeReference<List<TaskDef>>() {};
+
+    public void loadTaskDefs(String resourcePath) throws IOException {
+        InputStream resource = WorkflowExecutor.class.getResourceAsStream(resourcePath);
+        if(resource != null) {
+            List<TaskDef> taskDefs = om.readValue(resource, listOfTaskDefs);
+            loadMetadata(taskDefs);
+        }
+    }
+
+    public void loadWorkflowDefs(String resourcePath) throws IOException {
+        InputStream resource = WorkflowExecutor.class.getResourceAsStream(resourcePath);
+        if(resource != null) {
+            WorkflowDef workflowDef = om.readValue(resource, WorkflowDef.class);
+            loadMetadata(workflowDef);
+        }
+    }
+
+    public void loadMetadata(WorkflowDef workflowDef) {
+        metadataClient.registerWorkflowDef(workflowDef);
+    }
+
+    public void loadMetadata(List<TaskDef> taskDefs) {
+        metadataClient.registerTaskDefs(taskDefs);
+    }
+
+    private void scanWorkers(String basePackage) {
+        try {
+            List<String> packagesToScan = new ArrayList<>();
+            if(basePackage != null) {
+                String[] packages = basePackage.split(",");
+                for(String pkg : packages) {
+                    packagesToScan.add(pkg);
+                }
+            }
+
+            long s = System.currentTimeMillis();
+            ClassPath.from(WorkflowExecutor.class.getClassLoader()).getAllClasses().stream().forEach(classMeta -> {
+                String name = classMeta.getName();
+                if(!includePackage(packagesToScan, name)) {
+                    return;
+                }
+                try {
+                    Class<?> clazz = classMeta.load();
+                    Object obj = clazz.getConstructor().newInstance();
+                    scanClass(clazz, obj);
+                }catch(Throwable t) {
+
+                }
+            });
+            long e = System.currentTimeMillis();
+            System.out.println("Took " + (e-s) + " ms to scan all the classes");
```

Comment:
Again should consider logger instead of system out print

Reply:
Made this change